src/platforms/javascript/common/install/index.mdx

```diff
@@ -20,3 +20,35 @@ notSupported:
 ---
 
 <PageGrid />
+
+## When to use which Installation Method
+
+Depending on the concrete needs of your application, you may wonder which installation method you should use.
+
+### Why to use the Loader Script over NPM
+
+Using the Loader Script is the easiest way to start using Sentry. You only need to add a script tag to your application, and we'll take care of setting everything up correctly for you.
+
+- **Easy to set up**: You only need to [add a script tag](./loader/#using-the-loader) to your application.
+
+- **Easy to upgrade**: We'll take care of always shipping the latest version of the SDK to your users - no action needed on your side.
+
+- **Easy to add additional configuration**: You can still [configure the SDK](./loader/#sdk-configuration) to your liking via `Sentry.onLoad()`.
+
+- **Lazy-loading**: If you only want to use Sentry for errors, we'll only load the SDK when an error occurs. This means that we can reduce the amount of JavaScript loaded on your page until an error occurs.
```

Comment:
Should we say directly 'reduce bundle size'. That is what this is, right?

Reply:
I added this sentence:
> This reduces the initial bundle size.